install.sh

```diff
@@ -1048,7 +1048,7 @@ EOS
 else
   cat <<EOS
 - Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
-    (echo; echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"') >> ${shell_rcfile}
+    printf '%s\n' '' 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"' >> ${shell_rcfile}
```

Comment:
Wait, this will print
```
printf '%s\n' '' 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"'
```
but we want `${HOMEBREW_PREFIX}` to be expanded.

Reply:
(Yes, I know it's a bit confusing, but) We're in a `cat <<EOS` context so single quotes don't really mean non-expansion. See https://github.com/Homebrew/install/actions/runs/11034527267/job/30649515010?pr=881#step:8:489.

Reply:
Right, yes, gotcha!